backend/src/main/java/de/neuefische/backend/service/TimerService.java

```diff
@@ -0,0 +1,35 @@
+package de.neuefische.backend.service;
+
+import de.neuefische.backend.model.TimerModel;
+import de.neuefische.backend.repo.TimerRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+@Service
+public class TimerService {
+
+    private final TimerRepo timerRepo;
+
+    @Autowired
+    public TimerService(TimerRepo timerRepo) {
+        this.timerRepo = timerRepo;
+    }
+
+    public TimerModel getTimer() {
+        Optional<TimerModel> optionalTimer = timerRepo.findFirstByOrderById();
+
+        if (optionalTimer.isEmpty()){
+            throw new NoSuchElementException("No timer in database");
```

Comment:
Maybe it is more precise to use a custom Exception

Reply:
I changed it to an IllegalStateException which should be more precise in this case.